core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowBasePlugin.scala

```diff
@@ -56,7 +56,8 @@ object CloudflowBasePlugin extends AutoPlugin {
           // this artifact needs to have `%` and not `%%` as we build the runner jar
           // without version information. This is required for Flink runtime as a fixed name
           // jar needs to be uploaded to a specific location for Flink operator to pick up
-          "com.lightbend.cloudflow" % "cloudflow-runner" % BuildInfo.version
+          "com.lightbend.cloudflow" % "cloudflow-runner"       % BuildInfo.version,
+          "com.lightbend.cloudflow" %% "cloudflow-localrunner" % BuildInfo.version
```

Comment:
Does this mean that localrunner and all of its dependencies land up in the image that is built?

Reply:
Yes, but...
```
lazy val localRunner =
  cloudflowModule("cloudflow-localrunner")
    .enablePlugins(BuildInfoPlugin, ScalafmtPlugin)
    .dependsOn(streamlets, blueprint)
    .settings(
      scalafmtOnCompile := true
    )
```
I removed all dependencies from the build and, in particular, moved the log dependencies to the runtime.